build-release-tools/application/release_to_atlas.py

```diff
@@ -236,9 +105,14 @@ def upload_boxs(build_directory, atlas, is_release, atlas_version):
                 atlas_version = full_file_path.split('/')[-1:][0].strip(".box").split('-')[3]
             else:
                 from datetime import datetime
-                datatime_now_md = datetime.utcnow().strftime("0.%m.%d")
+                import time
```

Comment:
I'm not sure(forgive me), is import in the middle of python code a good practice ?